OrbitLinuxTracing/UprobesCallstackManager.h

```diff
@@ -0,0 +1,182 @@
+#ifndef ORBIT_LINUX_TRACING_UPROBES_CALLSTACK_MANAGER_H_
+#define ORBIT_LINUX_TRACING_UPROBES_CALLSTACK_MANAGER_H_
+
+#include "LibunwindstackUnwinder.h"
+#include "PerfEvent.h"
+#include "absl/container/flat_hash_map.h"
+
+namespace LinuxTracing {
+
+// LateUnwindCallstack holds either a UprobesWithStackPerfEvent and the snapshot
+// of the maps needed to unwind it, or the already unwound callstack.
+class LateUnwindCallstack {
+ public:
+  // Careful: UprobesWithStackPerfEvent* uprobes_event is moved from (a copy
+  // would be expensive).
+  explicit LateUnwindCallstack(UprobesWithStackPerfEvent* uprobes_event,
```

Comment:
Should UprobesWithStackPerfEvent be a rvalue? To make it clear it is moved from.